content/en/opentelemetry/agent/migration.md

```diff
@@ -0,0 +1,390 @@
+Here are two example Collector configuration files:
+
+{{< tabs >}}
+{{% tab "Custom Collector components" %}}
+
+This example uses a custom `metricstransform` component in `collector-config.yaml`:
+
+{{< highlight yaml "hl_lines=17-21 36" >}}
+receivers:
+  otlp:
+    protocols:
+      grpc:
+         endpoint: 0.0.0.0:4317
+      http:
+         endpoint: 0.0.0.0:4318
+exporters:
+  datadog:
+    api:
+      key: ${env:DD_API_KEY}
+processors:
+  infraattributes:
+    cardinality: 2
+  batch:
+    timeout: 10s
+  metricstransform:
+    transforms:
+      - include: system.cpu.usage
+        action: update
+        new_name: system.cpu.usage_time
+connectors:
+  datadog/connector:
+    traces:
+      compute_top_level_by_span_kind: true
+      peer_tags_aggregation: true
+      compute_stats_by_span_kind: true
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [infraattributes, batch]
+      exporters: [datadog/connector, datadog]
+    metrics:
+      receivers: [otlp, datadog/connector]
+      processors: [metricstransform, infraattributes, batch]
+      exporters: [datadog]
+    logs:
+      receivers: [otlp]
+      processors: [infraattributes, batch]
+      exporters: [datadog]
+{{< /highlight >}}
+
+In this case, you need to follow [Use Custom OpenTelemetry Components with Datadog Agent][4].
+
+{{% /tab %}}
+
+{{% tab "Default Agent components" %}}
+
+This example only uses components included in the Datadog Agent by default:
+
+{{< code-block lang="yaml" filename="collector-config.yaml" disable_copy="true" collapsible="true" >}}
+receivers:
+  otlp:
+    protocols:
+      grpc:
+         endpoint: 0.0.0.0:4317
+      http:
+         endpoint: 0.0.0.0:4318
+exporters:
+  datadog:
+    api:
+      key: ${env:DD_API_KEY}
+processors:
+  infraattributes:
+    cardinality: 2
+  batch:
+    timeout: 10s
+connectors:
+  datadog/connector:
+    traces:
+      compute_top_level_by_span_kind: true
+      peer_tags_aggregation: true
+      compute_stats_by_span_kind: true
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [infraattributes, batch]
+      exporters: [datadog/connector, datadog]
+    metrics:
+      receivers: [otlp, datadog/connector]
+      processors: [infraattributes, batch]
+      exporters: [datadog]
+    logs:
+      receivers: [otlp]
+      processors: [infraattributes, batch]
+      exporters: [datadog]
+{{< /code-block >}}
+
+In this case, you can proceed to installing the Agent with the embedded OpenTelemetry Collector.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Install the Agent with OpenTelemetry Collector
+
+Follow these steps to install the Agent with embedded OpenTelemetry Collector.
+
+### Add the Datadog Helm Repository
+
+To add the Datadog repository to your Helm repositories:
+
+```shell
+helm repo add datadog https://helm.datadoghq.com
+helm repo update
+```
+
+### Set up Datadog API and application keys
+
+1. Get the Datadog [API][2] and [application keys][3].
```

Comment:
These links don't go to API or application keys pages. Is that intentional?

Reply:
Ah, great catch! That was an error after moving some content around.